shared/enclave/server/handlers/displayAddress.ts

```diff
@@ -0,0 +1,7 @@
+import { getWalletLib } from 'shared/enclave/server/wallets';
+import { DisplayAddressParams, DisplayAddressResponse } from 'shared/enclave/types';
+
+export default function(params: DisplayAddressParams): Promise<DisplayAddressResponse> {
```

Comment:
Not really a fan of default exports especially for functions that'll be used often, as it makes refactor-by-reference (powered by ts) not possible as the default import name is user defined

Reply:
This goes for the functions below too